### PR TITLE
mergify: less updates

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

update the only PR with 3 or more approval so they are close to getting merged
@rohitgr7 shall we drop the label since it does not have any effect on the true merging?

## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified
 - [x] **Check that target branch and milestone match!**

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: .github/mergify.yml
Comment (Borda) [inline .github/mergify.yml:43]:
```suggestion
```
@rohitgr7 shall we drop the label since it does not have any effect on the true merging?

Comment (rohitgr7) [reply .github/mergify.yml:43]: then we need to ensure that auto-merge should be enabled if a PR is ready to go else if it isn't then there is a possibility even after all the required approvals the PR is not merged. At least this still gives an indication that ok PR is good to go and one can then just directly merge it.

Comment (Borda) [reply .github/mergify.yml:43]: well yes, we can return also the merge action to mergify, but still, I would make it on core responsibility to set the auto-merge when they feel that the PR is suitable to merge...

Comment (rohitgr7) [reply .github/mergify.yml:43]: ok my bad, not required here I guess.. I thought you are removing the label completely.

**Review by rohitgr7 — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]

File: .github/mergify.yml
Comment (Borda) [inline .github/mergify.yml:43]:
```suggestion
      - -title~=(?i)wip # skip all PR that title contains “WIP” (ignoring case)
      - "#approved-reviews-by>=3" # number of review approvals
```

**Review by tchaton — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

**Pull request merged**
